spacy/scorer.py

```diff
@@ -343,32 +350,39 @@ def score_spans(
             pred_spans = set()
             for span in getter(gold_doc, attr):
                 gold_span = (span.label_, span.start, span.end - 1)
+                if not consider_label:
+                    gold_span = (span.start, span.end - 1)
```

Comment:
I think an `if consider_label / else` would be clearer here (also in a few other spots).

Reply:
You think? Feels equally clear to me, and the current version is a bit more concise... But it doesn't matter much either way.

Reply:
No, it doesn't really matter, but it surprised me when I was reading through it to have it set twice. (I also go back and forth about this kind of thing.)